packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -207,32 +232,52 @@ If the expression makes up the entire attribute value, you can leave off the quo
 html`<img src="/images/${this.image}">`;
 ```
 
-### Setting attributes if data is defined { #ifDefined }
+Note, some primitive values are handled specially in attributes. Boolean values are converted to strings so, for example, `false` renders `'false'`. Both `undefined` and `null` render to an attribute as an empty string.
+
+### Boolean attributes {#boolean-attribute-expressions }
+
+To set a boolean attribute, use the `?` prefix with the attribute name. The attribute is added if the expression evaluates to a truthy value, removed if it evaluates to a falsy value:
+
+```js
+html`<div ?hidden=${!this.showAdditional}>This text may be hidden.</div>`;
+```
+
+### Removing an attribute { #removing-attribute }
+
+Sometimes you want to set an attribute only if the part's value(s) meet certain conditions, and otherwise remove the attribute. For common "boolean attributes" like `disabled` and `hidden` where you want to set the attribute to an empty string for a truthy value and remove it otherwise, use a [boolean attribute](#boolean-attribute-expressions). Sometimes, however, an attribute should be set to a specific value in some cases, and removed in others.
```

Comment:
I think "the parts values" is kind of confusing, because we don't really talk about parts until we get into custom directives. So maybe focus instead on the desired outcome?

```suggestion
Sometimes you want to set an attribute only under certain conditions, and otherwise remove the attribute. For common "boolean attributes" like `disabled` and `hidden` where you want to set the attribute to an empty string for a truthy value and remove it otherwise, use a [boolean attribute](#boolean-attribute-expressions). Sometimes, however, you might require a different condition for adding or removing an attribute. 
```